posthog/src/test/java/com/posthog/internal/PostHogFeatureFlagsV3Test.kt

```diff
@@ -21,8 +21,8 @@ import kotlin.test.assertTrue
 internal class PostHogFeatureFlagsV3Test {
```

Comment:
this file is called `PostHogFeatureFlagsV3Test.kt`, but v3 is from decide, should we rename it?